.gitignore

```diff
@@ -84,12 +99,13 @@ src/app/jni/output
 # Python build directory
 build/
 
-# MSVC build directories
```

Comment:
I think this block also has to stay, for portable projects at least. Probably `zproject_git.gsl` should take care of this (not a CZMQ issue except further regeneration of the project when that fix lands).